package ru.yandex.practicum.gym;

public class Group {
    //название группы
    private String title;
    //тип (взрослая или детская)
    private Age age;

    @Override
    public String toString() {
        return "Group{" +
                "title='" + title + '\'' +
                ", age=" + age +
                ", duration=" + duration +
                '}';
    }

    //длительность (в минутах)
    private int duration;

    public Group(String title, Age age, int duration) {
        this.title = title;
        this.age = age;
        this.duration = duration;
    }

    public String getTitle() {
        return title;
    }

    public Age getAge() {
        return age;
    }

    public int getDuration() {
        return duration;
    }
}
